.github/workflows/smalltalk-ci.yml

```diff
@@ -0,0 +1,21 @@
+name: CI
+
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the development branch
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
Do we want to continue testing on macOS and windows? My guess is yes

Reply:
Yes!

Reply:
Addressed in [this commit](https://github.com/PolyMathOrg/PolyMath/pull/183/commits/0ff00106d336bc7f2017653a8a5d24f77e3d4f07)